### PR TITLE
Add chromothripsis methods

### PR DESCRIPTION
**Pull request by LauraEgolf**
### Purpose
Add methods for chromothripsis analysis with ShatterSeek.

### Issue
#75

#### Which areas should receive a particularly close look?
N/A

#### Is there anything that you want to discuss further?
No

#### Is the pull request ready for review?
Yes

### Pull review checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [x] All figures follow the [Manubot figure instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#figures)
- [x] There are no new spelling errors identified by the [Manubot spellchecker](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#spellchecking)

### Spellcheck Step

The dictionary used for spellchecking can be updated.
Edit the file in `build/assets/custom-dictionary.txt` by adding new entries to the end.
You do not need to change anything else.
However, if you want to update the first line to have an accurate count of words and you want to remove non-unique ones, run the following command from within `build/assets` on your favorite OS X or Linux machine:
```
(( len = $(awk '!a[$0]++' < custom-dictionary.txt | wc -l ) - 1 )); tmpfile="$(mktemp)"; echo "personal_ws-1.1 en $len utf-8" > $tmpfile; tail -n +2 custom-dictionary.txt | awk '!a[$0]++' >> $tmpfile; mv $tmpfile custom-dictionary.txt
```

### CODE REVIEW

**Comment by LauraEgolf** [comment]
I can create a separate issue if necessary since this doesn’t fall under methods - but I fixed a typo in the legend for Figure 3. The adjusted R2 value on the plot of SV vs. CNV breaks is 0.19, but the text says 0.436. I changed the text to 0.19. (Cc @jharenza)

**Comment by jharenza** [comment]
> I can create a separate issue if necessary since this doesn’t fall under methods - but I fixed a typo in the legend for Figure 3. The adjusted R2 value on the plot of SV vs. CNV breaks is 0.19, but the text says 0.436. I changed the text to 0.19. (Cc @jharenza)

Ah, I think I wrote R in text, not R^2... maybe we should update the figure to use the cor coefficient for consistency across any corplots

**Comment by LauraEgolf** [comment]
> Ah, I think I wrote R in text, not R^2... maybe we should update the figure to use the cor coefficient for consistency across any corplots

Oh I see - I didn't catch on to the fact that you converted R^2 to R. I will revert that commit and update the plot.

**Comment by jharenza** [comment]
> > Ah, I think I wrote R in text, not R^2... maybe we should update the figure to use the cor coefficient for consistency across any corplots
> 
> 
> 
> Oh I see - I didn't catch on to the fact that you converted R^2 to R. I will revert that commit and update the plot.

No worries- I had in the back of my head to have this converted on the plot and forgot so thanks for catching and making an issue!

File: content/06.methods.md [hunk 1]
```diff
@@ -230,6 +230,13 @@ The hg38 reference for SV calling used was limited to canonical chromosome regio
 The somatic DNA workflow for SNV, indel, copy number, and SV calling can be found in the [KidsFirst Github repository](https://github.com/kids-first/kf-somatic-workflow).
 Manta SV output was annotated using [AnnotSV v2.1](https://lbgi.fr/AnnotSV/) [@doi:10.1093/bioinformatics/bty304] and the workflow can be found in the [D3b GitHub repository](https://github.com/d3b-center/annotSV/tree/master).
 
+#### Chromothripsis Analysis (WGS samples only)
+
+Candidate chromothripsis regions were identified in the set of independent tumor WGS samples with ShatterSeek [@doi:10.1038/s41588-019-0576-7], using consensus CNV calls and Manta SV calls that passed all filters. 
+Only chromosomes 1-22 and X were considered. 
+The consensus CNV data were modified to fit ShatterSeek input requirements: CNV-neutral or excluded regions (both annotated as `NA` in the consensus data) were filled in with the respective sample’s ploidy value from Control-FREEC, and consecutive segments with the same copy number value were merged. 
+Candidate chromothripsis regions were classified as high- or low-confidence by applying the statistical criteria described by the authors.
```
Comment (jaclyn-taroni) [inline content/06.methods.md:238]: I think making this a touch more explicit would be good!
```suggestion
Candidate chromothripsis regions were classified as high- or low-confidence by applying the statistical criteria described by the ShatterSeek authors.
```

File: content/06.methods.md [hunk 2]
```diff
@@ -230,6 +230,13 @@ The hg38 reference for SV calling used was limited to canonical chromosome regio
 The somatic DNA workflow for SNV, indel, copy number, and SV calling can be found in the [KidsFirst Github repository](https://github.com/kids-first/kf-somatic-workflow).
 Manta SV output was annotated using [AnnotSV v2.1](https://lbgi.fr/AnnotSV/) [@doi:10.1093/bioinformatics/bty304] and the workflow can be found in the [D3b GitHub repository](https://github.com/d3b-center/annotSV/tree/master).
 
+#### Chromothripsis Analysis (WGS samples only)
+
+Candidate chromothripsis regions were identified in the set of independent tumor WGS samples with ShatterSeek [@doi:10.1038/s41588-019-0576-7], using consensus CNV calls and Manta SV calls that passed all filters.
```
Comment (jaclyn-taroni) [inline content/06.methods.md:235]: This ordering is less ambiguous re: filters in my opinion:
```suggestion
Candidate chromothripsis regions were identified in the set of independent tumor WGS samples with ShatterSeek [@doi:10.1038/s41588-019-0576-7], using Manta SV calls that passed all filters and consensus CNV calls. 
```

**Review by jaclyn-taroni — APPROVED** [review]
Looks good to me 👍🏻  I only had a couple wording comments for clarity.  Once those are addressed, I'll get this merged. Thank you @LauraEgolf!

**Pull request merged**